Source/Diagnostics/ParticleDiag/ParticleDiag.cpp

```diff
@@ -0,0 +1,71 @@
+#include "WarpX.H"
+#include "ParticleDiag.H"
+// #include "Utils/WarpXUtil.H"
+
+#include <AMReX_ParmParse.H>
+
+using namespace amrex;
+
+ParticleDiag::ParticleDiag(std::string diag_name, std::string name, WarpXParticleContainer* pc)
+    : m_diag_name(diag_name), m_name(name), m_pc(pc)
+{
+    ParmParse pp(diag_name + "." + name);
+    if (!pp.queryarr("variables", variables)){
+        variables = {"Ex", "Ey", "Ez", "Bx", "By", "Bz", "ux", "uy", "uz", "w"};
+    }
+
+    //variable to set plot_flags size
+    int plot_flag_size = PIdx::nattribs;
+    if(WarpX::do_back_transformed_diagnostics && m_pc->doBackTransformedDiagnostics())
+        plot_flag_size += 6;
```

Comment:
maybe comment which additional attributes are counted here

Reply:
Good point, I'l add a comment in the next commit.